scripts/us_census/pep/monthly_population_estimate/input_url.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/tables/2000-2009/state/totals/na-est2009-01.csv"
+    },
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/tables/2020-2021/national/totals/NA-EST2021-POP.xlsx"
+    },
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/tables/2010-2019/national/totals/na-est2019-01.xlsx"
+    },
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/tables/1990-2000/national/totals/nat-total.txt"
+    },
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/tables/2020-2023/national/totals/NA-EST2023-POP.xlsx"
```

Comment:
Remove this link as it should be dynamically constructed using the future year URL.

Reply:
Removed @krishnaswamypradeep